tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -1062,21 +1091,15 @@ inline bool VerifyConstants(RemapperContext* ctx,
   return true;
 }
 
-// Gelu in python api generates a number of nodes in the graph. Depending on the
-// parmeter `approximate={True/False}` different types of ops are generated. We
-// distinguish them as `GeluExact` that uses Erf and `GeluApproximate` that
-// uses Tanh.
-bool FindMatMulBiasAddAndGelu(RemapperContext* ctx, int node_index,
-                              std::map<string, int>* matched_nodes_map,
-                              std::set<int>* remove_node_indices,
-                              bool* is_gelu_approximate) {
-  // Gelu fusion is enabled with oneDNN or cublasLt library.
-  if (!IsMKLEnabled() && !BlasLtMatmulEnabled()) return false;
-
+bool IsMatchedMatMulBiasAddAndGeluExact(
+    RemapperContext& ctx, int node_index,
+    std::map<string, int>* matched_nodes_map,
+    std::set<int>* remove_node_indices) {
```

Comment:
It looks like these could have default values of `nullptr`? It would simplify one of the calls below.

Reply:
Done.